src/main/java/org/cactoos/map/Sticky.java

```diff
@@ -158,12 +158,14 @@ public Sticky(final Map<X, Y> map,
      */
     public Sticky(final Map<X, Y> map) {
```

Comment:
@andreoss this class should be removed too as discussed in the ticket

Reply:
@victornoel Done.